ctapipe/containers.py

```diff
@@ -57,13 +57,65 @@
     "StatisticsContainer",
     "IntensityStatisticsContainer",
     "PeakTimeStatisticsContainer",
+    "SchedulingBlockContainer",
+    "ObservationBlockContainer",
+    "ObservingMode",
+    "ObservationBlockState",
 ]
 
 
 # see https://github.com/astropy/astropy/issues/6509
 NAN_TIME = Time(0, format="mjd", scale="tai")
 
 
+class SchedulingBlockType(enum.Enum):
+    """
+    Types of Scheduling Block
+    """
+
+    UNKNOWN = -1
+    OBSERVATION = 0
+    CALIBRATION = 1
+    ENGINEERING = 2
+
+
+class ObservationBlockState(enum.Enum):
+    """
+    Observation Block State, from XXX
+    """
+
+    UNKNOWN = -1
+    FAILED = 0
+    COMPLETED_SUCCEDED = 1
+    COMPLETED_CANCELED = 2
+    COMPLETED_TRUNCATED = 3
+    ARCHIVED = 4
+
+
+class ObservingMode(enum.Enum):
+    """How a scheduling block is observed."""
+
+    UNKNOWN = -1
+    WOBBLE = 0
+    ON_OFF = 1
+    GRID = 2
+    CUSTOM = 3
+
+
+class PointingMode(enum.Enum):
+    UNKNOWN = -1
+    TRACK = 0  # track a fixed point that moves with the sky
```

Comment:
If you make these comments like this:

```
#: track a fixed point in ICRS
TRACK = 0
```

they are picked up by sphinx as docstrings, see: 
https://github.com/cta-observatory/ctapipe/blob/8c42f54f805d81b152821360f07974d17acbd7f2/ctapipe/instrument/optics.py#L23